src/components/teamBoxes/TeamTwoBox.jsx

```diff
@@ -1,30 +1,25 @@
 import React, { useEffect } from "react";
 import "./blueTeamBox.css";
 import { useDispatch, useSelector } from "react-redux";
-import { child, get, onValue, ref, set } from "firebase/database";
-import { database } from "../../utils/firebase";
-import {
-  setTeamTwoOperatives,
-  setTeamTwoSpymaster,
-} from "../../store/teamTwoSlice";
-
-const BlueTeamBox = () => {
-  const dispatch = useDispatch();
-  const { playerId, roomId, username } = useSelector((state) => state.player);
-
-  const teamTwoOperativesRef = ref(
-    database,
-    `rooms/${roomId}/team-2/operatives/`
-  );
-  const teamTwoSpymasterRef = ref(
-    database,
-    `rooms/${roomId}/team-2/spymaster/`
-  );
+import "./redTeamBox.css";
```

Comment:
Looks like this component didnt take the changes in from main, eg) we dont need the redTeamBox.css for this .

Maybe the best solution here is just to solve the eslint errors you see when you open this file, and we can just push those changes!